lib/bookingsync/engine.rb

```diff
@@ -67,21 +75,26 @@ def self.standalone!
     # The ENV variables used for configuration are described in {file:README.md}.
     #
     # @return [OAuth2::Client] configured OAuth client
-    def self.oauth_client
+    def self.oauth_client(client_id: ENV['BOOKINGSYNC_APP_ID'], client_secret: ENV['BOOKINGSYNC_APP_SECRET'])
       client_options = {
         site: ENV['BOOKINGSYNC_URL'] || 'https://www.bookingsync.com',
         connection_opts: { headers: { accept: "application/vnd.api+json" } }
       }
       client_options[:ssl] = { verify: ENV['BOOKINGSYNC_VERIFY_SSL'] != 'false' }
-      OAuth2::Client.new(ENV['BOOKINGSYNC_APP_ID'], ENV['BOOKINGSYNC_APP_SECRET'],
-        client_options)
+      OAuth2::Client.new(client_id, client_secret, client_options)
+    end
+
+    def self.application_token(client_id: nil, client_secret: nil)
+      oauth_client(client_id: client_id, client_secret: client_secret).client_credentials.get_token
     end
 
-    def self.application_token
-      oauth_client.client_credentials.get_token
+    def self.support_multi_applications?
+      ENV['BOOKINGSYNC_ENGINE_SUPPORT_MULTI_APPLICATIONS'].to_s.downcase == "true"
```

Comment:
single quotes -> double quotes

maybe Rubocop will deal with that automatically?